example/quora-duplicate-question-pair.rb

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env ruby
+
+require "datasets"
+
+question_pair = Datasets::QuoraDuplicateQuestionPair.new
+question_pair.each do |pair|
+  p [
+    pair.id,
+    pair.first_question_id,
+    pair.second_question_id,
+    pair.first_question,
+    pair.second_question,
+    pair.duplicated
```

Comment:
How about adding `pair.duplicated?` API because this returns a boolean value?

We can implement it something like the following

```ruby
class Record < Struct.new(:id,
                          :first_question_id,
                          :second_question_id,
                          :first_question,
                          :second_question,
                          :duplicated)
  alias_method :duplicated?, :duplicated
end
```

See also:
*  https://github.com/red-data-tools/red-datasets/blob/master/lib/datasets/postal-code-japan.rb#L8-L33
* https://github.com/red-data-tools/red-datasets/blob/master/lib/datasets/sudachi-synonym-dictionary.rb#L7-L17

Reply:
fix: [6b6c0cd](https://github.com/red-data-tools/red-datasets/pull/149/commits/6b6c0cdca99a2575ddf6aede194ee52728bae78b) Thank you! That sounds pretty nice because I think users will expect it too.

Reply:
```suggestion
    pair.duplicated?
```